mentor-moments.md

```diff
@@ -0,0 +1,11 @@
+# Mentor moments
+
+During the course, each attendee is assigned a mentor who is working as a web developer. During the end of the course (week 8?) the attendees have the opportunity to chat with the mentor about topics and questions they are interested in, for example:
+
+- What do you do at work?
+- What salary can one expect as a entry level junior developer?
+- How can I find my first project?
+- What are topics one could specialize in (BE/FE/UX)?
+
+List of mentors available: 
+- Annie Example, BE developer at Example Company
```

Comment:
I suggest changing to "BE developer in Example type of company" or similar anonymized since this is a public repo. We can later fix it with a secure link or maybe add the names with consent of the mentors.